### PR TITLE
implementasi real data dashboard

### PR DESCRIPTION
**Pull request by mosleim**
related to : [this](https://trello.com/c/ZGUAJ1wd/11-backend-implement-real-data-dashboard-admin)

Implement Real Data Dashboard Admin

This task sepately is done. Except updating icon on blade template.
I don't have idea to writing code in blade templating.

### CODE REVIEW

File: app/Http/Controllers/DashboardController.php [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
+use App\Company;
+use App\User;
 class DashboardController extends Controller
 {
     public function index() {
-        return view('dashboard.index');
+        
+        return view('dashboard.index', [
+            'postCount' => Post::count(),
+            'companyCount'=>Company::count(),
```
[the remark this answers is not in the document]
Comment (adeyahya) [reply app/Http/Controllers/DashboardController.php:15]: I think this is a minor thing, can we solve this problem using pre-commit fmt?

[the remark this answers is not in the document]
Comment (adeyahya) [reply app/Http/Controllers/DashboardController.php:15]: @dns2012  I think we should use prettier and add pre-commit hooks using husky, so the contributor just focus to the logic, the formatter part will be automatically done by the script.

[the remark this answers is not in the document]
Comment (adeyahya) [reply app/Http/Controllers/DashboardController.php:15]: I think ill create a PR for this part.

[the remark this answers is not in the document]
Comment (mosleim) [reply app/Http/Controllers/DashboardController.php:15]: @adeyahya agree with you!
My IDE not configured for PHP Development. No linter here.  I don't know the last version of PSR now. I think, we need some tools to achieve standart of code.

[the remark this answers is not in the document]
Comment (adeyahya) [reply app/Http/Controllers/DashboardController.php]: My previous pr about commit hooks doesnt require specific ide setup

**Pull request merged**